src/global.css

```diff
@@ -67,6 +67,16 @@
   /* custom animations */
   --animate-pulse-scale: pulse-scale 5s ease-in-out infinite;
   --animate-fade-scale: fade-scale 5s ease-in-out infinite;
+
+  --animate-carousel: keyframe-carousel 20s infinite linear;
+  @keyframes keyframe-carousel {
+    0% {
+      transform: translateX(0);
+    }
+    100% {
+      transform: translateX(-33.3333%);
+    }
+  }
```

Comment:
Move it after line 248 along with the other animations.

Reply:
It has to be in the `@theme` block according to the Tailwind docs

Reply:
I put them outside and worked, but the comment is more about grouping them, so we could move the other keyframes here..